pkg/tso/keyspace_group_manager.go

```diff
@@ -606,7 +607,7 @@ func (kgm *KeyspaceGroupManager) primaryPriorityCheckLoop() {
 	defer kgm.wg.Done()
 
 	failpoint.Inject("fastPrimaryPriorityCheck", func() {
-		kgm.primaryPriorityCheckInterval = 200 * time.Millisecond
+		kgm.primaryPriorityCheckInterval = 1 * time.Second
```

Comment:
Why enlarge it?

Reply:
I feel no need to keep calling `TransferPrimary` every 200ms, which probably won't happen in a real scenario?

Reply:
Is it used to accelerate the speed of the test?

Reply:
got it, and have reverted it :)